Detectors/ITSMFT/ITS/simulation/include/ITSSimulation/Detector.h

```diff
@@ -26,12 +26,12 @@ namespace o2 { namespace ITSMFT { class Point; }}  // lines 22-22
 namespace o2 { namespace ITS { class GeometryHandler; }}
 namespace o2 { namespace ITS { class MisalignmentParameter; }}
 namespace o2 { namespace ITS { class GeometryTGeo; }}
-namespace o2 { namespace ITS { class V1Layer; }}  // lines 23-23
+namespace o2 { namespace ITS { class V3Layer; }}  // lines 23-23
```

Comment:
ops, sorry, I thought something more C++ in-deep :)
I took out all obsolete/out of sync comments.